### ISSUE TITLE
Sound?

### ISSUE THREAD

**Issue by YanikaZerafa** [lggauci/IMTAssignment2016#11]
When I was playing the demo, I realized that there is no sound. Will you implement the sound in the full game? And if so what sound effects will you implement?

**Comment by lggauci** [comment]
Yes there will be several sound effects, some of which would be implemented when the player collect score items and powerups, when the user jumps and when he interacts with other objects. Also  background music will be found in each level
